package/src/components/Message/hooks/useMessageActionHandlers.ts

```diff
@@ -84,6 +93,18 @@ export const useMessageActionHandlers = <
     setEditingState(message);
   };
 
+  const handleFlagMessage = async () => {
+    try {
+      await client.flagMessage(message.id);
+      Alert.alert(t('Message flagged'), t('The message has been reported to a moderator.'));
```

Comment:
Is there a way to let the client decide how these messages are displayed? Using `Alert` from `react-native` might be inconsistent with the client UI for alerts and modals.

Reply:
Hey @jakubdrozdek, right now we don't have an option to customize this. How important would it be for you?